eng/targets/Npm.Common.targets

```diff
@@ -20,6 +20,12 @@
 
   <Target Name="_CheckForInvalidConfiguration">
     <Error Text="Missing expected property: PackageId" Condition="'$(IsPackable)' != 'false' and '$(PackageId)' == ''" />
+
+    <Exec ContinueOnError="true" Command="node -v">
+       <Output TaskParameter="ExitCode" PropertyName="ErrorCode"/>
+    </Exec>
+
+    <Error Text="Building *.npmproj but NodeJS was not detected on path. Ensure NodeJS is on path or disable building NodeJS projects with /p:BuildNodeJs=false." Condition="'$(ErrorCode)' != '0'"/>
```

Comment:
Please add “If BuildNodeJs is set to false, the dependent C# projects will also be skipped. This includes projects in Components, Mvc, Templates, and Analyzers.” to the error message so people know what the consequences are.

Reply:
Hmm I could consider adding it here but I did add a warning "Some managed projects that depend on NodeJS projects will be skipped since building NodeJS is disabled."